dropwizard-testing/src/main/java/io/dropwizard/testing/junit/DropwizardAppRule.java

```diff
@@ -127,6 +136,20 @@ public DropwizardAppRule(Class<? extends Application<C>> applicationClass,
 
     public DropwizardAppRule(DropwizardTestSupport<C> testSupport) {
         this.testSupport = testSupport;
+        this.testSupport.addListener(new DropwizardTestSupport.ServiceListener<C>()
+        {
+            @Override
+            public void onRun(C configuration, Environment environment, DropwizardTestSupport<C> rule) throws Exception
+            {
+                environment.jersey().register(new LoggingExceptionMapper<Throwable>()
```

Comment:
If another `ExceptionMapper` captures it first it will do whatever the original exception mapper did.  This one will only trigger for uncaught exceptions.

The services I already have, and testing using this, are working fine.

Reply:
Excellent! Then, if it's not to much to ask, can you write a unit test where you register a custom `new LoggingExceptionMapper<Throwable>` (ie. return a 500 error with response of like "server is down :(") and assert that your listener picks up the exception, and the client still sees our custom "server is down" response?

Reply:
owww, sorry, now I get what you mean.

So, if you add a custom `new LoggingExceptionMapper<Throwable>`, the one include on test support won't trigger....

This only collects uncaught exceptions.  I handle a few exceptions as 400 and when I do that nothing is collected here.

Reply:
As long as custom `new LoggingExceptionMapper<Throwable>` overrides this one here, then this PR doesn't break backwards compatibility 👍 

With that, is there anything left you'd like to address before I give this PR another look?

Reply:
Na, I'm happy with it.